parachain/src/wasm_executor/mod.rs

```diff
@@ -166,7 +184,7 @@ pub fn validate_candidate_internal(
 ) -> Result<ValidationResult, ValidationError> {
 	let executor = sc_executor::WasmExecutor::new(
 		#[cfg(not(any(target_os = "android", target_os = "unknown")))]
-		sc_executor::WasmExecutionMethod::Compiled,
+		sc_executor::WasmExecutionMethod::Interpreted,
```

Comment:
may I ask why it was changed?

not sure how much of that code will be retained given https://github.com/paritytech/polkadot/pull/1609

Reply:
I had compilation issue. Will revert it to see

Reply:
In the previous backport I had to remove the feature wasmtime because the CI failed like this: https://gitlab.parity.io/parity/polkadot/-/jobs/635768

And now it's failing because of:

```
error[E0599]: no variant or associated item named `Compiled` found for enum `WasmExecutionMethod` in the current scope
   --> parachain/src/wasm_executor/mod.rs:187:37
    |
187 |         sc_executor::WasmExecutionMethod::Compiled,
    |                                           ^^^^^^^^ variant or associated item not found in `WasmExecutionMethod`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0599`.
error: could not compile `polkadot-parachain`.
```

So I talked to @bkchr and apparently I can ignore the failing test for now